examples/data-objects/multiview/container/src/container.tsx

```diff
@@ -23,17 +23,20 @@ const simpleCoordinateComponentId = "simpleCoordinate";
 const triangleCoordinateComponentId1 = "triangle1";
 const triangleCoordinateComponentId2 = "triangle2";
 const triangleCoordinateComponentId3 = "triangle3";
-const constellationComponentId = "constellation";
+const constellationComponentName = "constellation";
 
 const registryEntries = new Map([
     Coordinate.getFactory().registryEntry,
     Constellation.getFactory().registryEntry,
 ]);
 
 // Just a little helper, since we're going to create multiple coordinates.
-const createAndAttachCoordinate = async (runtime: IContainerRuntime, id: string) => {
+const createAndAttachCoordinate = async (runtime: IContainerRuntime, name: string) => {
+    const dataStore = await runtime.createDataStore(Coordinate.ComponentName);
```

Comment:
Coordinate.ComponentName as used here is not component name, it's component type, we should not confuse them and use right names. Something like Coordinate.getFactory().type is usually safe way to refer to type (assuming we do not overwrite it in registry